feature/accounts/src/commonMain/kotlin/org/mifos/mobile/feature/accounts/transactionDetail/TransactionDetailsScreen.kt

```diff
@@ -0,0 +1,361 @@
+
+            Text(
+                text = stringResource(Res.string.feature_transaction_detail_breakdown),
+                style = MifosTypography.labelLargeEmphasized,
+                modifier = Modifier.padding(bottom = 8.dp),
+            )
+
+            if (transaction.principal != null && transaction.principal > 0) {
+                DetailItem(
+                    label = stringResource(Res.string.feature_transaction_detail_principal),
+                    value = CurrencyFormatter.format(
+                        transaction.principal,
+                        transaction.currency,
+                        maximumFractionDigits = 2,
+                    ),
+                )
+            }
+            if (transaction.interest != null && transaction.interest > 0) {
+                DetailItem(
+                    label = stringResource(Res.string.feature_transaction_detail_interest),
+                    value = CurrencyFormatter.format(
+                        transaction.interest,
+                        transaction.currency,
+                        maximumFractionDigits = 2,
+                    ),
+                )
+            }
+            if (transaction.fees != null && transaction.fees > 0) {
+                DetailItem(
+                    label = stringResource(Res.string.feature_transaction_detail_fees),
+                    value = CurrencyFormatter.format(
+                        transaction.fees,
+                        transaction.currency,
+                        maximumFractionDigits = 2,
+                    ),
+                )
+            }
+            if (transaction.penalties != null && transaction.penalties > 0) {
+                DetailItem(
+                    label = stringResource(Res.string.feature_transaction_detail_penalties),
+                    value = CurrencyFormatter.format(
+                        transaction.penalties,
+                        transaction.currency,
+                        maximumFractionDigits = 2,
+                    ),
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun DetailItem(
+    label: String,
+    value: String,
+    modifier: Modifier = Modifier,
+    valueColor: Color = MaterialTheme.colorScheme.onSurface,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(vertical = DesignToken.padding.small),
+        horizontalArrangement = Arrangement.SpaceBetween,
+    ) {
+        Text(
+            text = label,
+            style = MifosTypography.bodyMedium,
+            color = MaterialTheme.colorScheme.onSurfaceVariant,
+        )
+        Text(
+            text = value,
+            style = MifosTypography.bodyMediumEmphasized,
+            color = valueColor,
+        )
+    }
+    HorizontalDivider(color = MaterialTheme.colorScheme.outlineVariant.copy(alpha = 0.5f))
+}
+
+@DevicePreview
+@Composable
+@Suppress("UnusedPrivateMember")
+private fun TransactionDetailContentPreview() {
```

Comment:
Private can be removed as done in other modules.

```suggestion
fun TransactionDetailContentPreview() {
```